src/treectl.c

```diff
@@ -836,7 +862,22 @@ ReadDirLevel(
       }
       else
       {
-          bFound = WFFindNext(&lfndta); // get it from dos
+          while (TRUE)
+          {
+              bFound = WFFindNext(&lfndta); // get it from dos
+              if (bFound)
+              {
+                  //  If it's a junction and we're not displaying those,
+                  //  loop again for the next entry
+                  if (lfndta.fd.dwFileAttributes & ATTR_JUNCTION &&
+                      (!(dwAttribs & ATTR_JUNCTION)))
+                  {
+
+                      continue;
+                  }
+              }
+              break;
+          }
       }

```

Comment:
This block and the one above at line 676 are doing essentially the same thing.  Can we create a method to "SkipJunctionPoints" or something like that?

Reply:
Done.